plugins/queue/mmap/branchmark_test.go

```diff
@@ -0,0 +1,109 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package mmap
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/apache/skywalking-satellite/internal/pkg/plugin"
+	"github.com/apache/skywalking-satellite/plugins/queue/api"
+)
+
+type benchmarkParam struct {
+	segmentSize      int
+	message          int // unit KB
+	maxInMemSegments int
+}
+
+var params = []benchmarkParam{
+	{segmentSize: 1024 * 128, message: 8, maxInMemSegments: 10},
+	// compare the influence of the segmentSize.
+	{segmentSize: 1024 * 256, message: 8, maxInMemSegments: 10},
+	// compare the influence of the maxInMemSegments.
+	{segmentSize: 1024 * 128, message: 8, maxInMemSegments: 20},
+	// compare the influence of the message size.
+	{segmentSize: 1024 * 128, message: 16, maxInMemSegments: 10},
+}
+
+func cleanBenchmarkQueue(b *testing.B, q api.Queue) {
+	if err := os.RemoveAll(q.(*Queue).QueueDir); err != nil {
+		b.Errorf("cannot remove test queue dir, %v", err)
+	}
+}
+
+func BenchmarkPush(b *testing.B) {
```

Comment:
As the benchmark, SkyWalking usually posts the results with specifications of the Mac.

Reply:
I can't see this has been fixed, but the comments have been resolved. Could you update?